src/core/manager.c

```diff
@@ -2815,6 +2814,15 @@ static int manager_dispatch_signal_fd(sd_event_source *source, int fd, uint32_t
                         /* This is a nop on init */
                         break;
 
+                case 25:
+                        if (MANAGER_IS_USER(m)) {
+                                m->objective = MANAGER_REEXECUTE;
+                                return 0;
+                        }
+
+                        /* This is a nop on PID1 */
```

Comment:
Hmm, I don't think it makes sense to make this a noop. We're reserving the number either way, and it seems better to bind to a restart (for consistency) than a noop.